Dockerfile

```
@@ -1,8 +1,8 @@
-FROM alpine:3.6
+FROM alpine:3.14
 
 ADD flaskapp.py /root/flaskapp.py
 ADD requirements.txt /root/requirements.txt
-RUN apk add --no-cache build-base openssl-dev libffi-dev python3 python3-dev py3-pip
+RUN apk add --no-cache build-base openssl-dev libffi-dev python3 python3-dev py3-pip cargo
```

Try this instead, it results in a faster build and a smaller container:

```suggestion
RUN apk add --no-cache build-base python3 py3-wheel py3-pip py3-cryptography
```